recipes-connectivity/libwebsockets/libwebsockets_3.0.0.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Canonical libwebsockets.org websocket library"
+HOMEPAGE = "https://libwebsockets.org/"
+LICENSE = "LGPL-2.1"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=4ce87f3facb6f911c142c8bef9bfb380"
+
+DEPENDS = "zlib"
+
+SRC_URI = "https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz"
+SRC_URI[md5sum] = "2a78c8c49bbbfe67c24637398d989958"
+SRC_URI[sha256sum] = "a6b611c212c52f161f70556339fdaa199b7e9b6a167c4638e086d19db75d6290"
+inherit cmake pkgconfig
+
+PACKAGECONFIG ?= "libuv client server http2 ssl"
```

Comment:
Maybe if you drop 'ssl', then the openssl stuff would go away?  'http2' and maybe 'client' could also be dropped.